docs/eventing/samples/writing-event-source-easy-way/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: "Writing an Event Source using the easy way"
-linkTitle: "Event Source - easy way"
+title: "Writing an Event Source using ContainerSource"
```

Comment:
Might bit a bit long for the linkTitle (since there is none, title is also used as a linkTile).

Reply:
Looks OK to me? Pretty sure we have longer ones :stuck_out_tongue:

Reply:
indeed: `Enabling requests to Knative services when additional authorization policies are enabled`. It just does not look good on the sidebar.

Reply:
@lionelvillard is there any action to complete here?